store/src/java/com/zimbra/cs/account/ldap/LdapLockoutPolicy.java

```diff
@@ -201,7 +206,28 @@ public void failedLogin(String protocol, String password) throws ServiceExceptio
         failedLogin(failedLogins, protocol, password);
     }
 
+    private void setCaptchaCount(String action ) {
```

Comment:
The type of the argument can be boolean if action is only "update" or "reset".

Reply:
Please rename method something like `updateCaptchaCount` with a boolean argument - say `failedCaptcha`.
Then instead of `if (action.equals("update"))` you can use `if (failedCaptcha)`